src/index.tsx

```diff
@@ -1,10 +1,7 @@
 import ReactDOM from 'react-dom';
-import { HashRouter } from 'react-router-dom';
-import { App } from './App';
+import { LayoutRouter } from './components/LayoutRouter';
 
 ReactDOM.render(
-  <HashRouter>
-    <App />
-  </HashRouter>,
+  <LayoutRouter />,
```

Comment:
It's not the correct way to structure your app.
Use a router inside your APP component. And use the app as the root of your application.
For layout create a `Layout` component that will render the view part of your application

```
  <Navigation />
  <main className="section">
    <Outlet />
  </main>
```